backend-lib/src/main/java/cash/z/ecc/android/sdk/internal/jni/RustBackend.kt

```diff
@@ -442,11 +442,13 @@ class RustBackend private constructor(
         }
 
     override suspend fun addProofsToPczt(pczt: ByteArray): ByteArray =
-        addProofsToPczt(
-            pczt,
-            spendParamsPath = saplingSpendFile.absolutePath,
-            outputParamsPath = saplingOutputFile.absolutePath,
-        )
+        withContext(SdkDispatchers.DATABASE_IO) {
```

Comment:
For my understanding: why does this need this synchronization context? It does not use the wallet database in any way.

Reply:
That's a good point. We just aligned this new API with the rest of the BE functions, but this is possibly the first BE API that does not need the DB path, so the context should not have any impact. I'm reverting this change.